locales/en/product-registration.json

```
@@ -0,0 +1,32 @@
+{
+  "register": "REGISTER",
+  "productImage": "Product Image(s)",
+  "addThreeImages": "Add up to 3 images",
+  "name": "Name",
+  "price": "Price (THB)",
+  "color": "Color",
+  "size": "Size",
+  "category": "Category",
+  "deliveryOption": "Delivery Option",
+  "footwear": "Footwear",
+  "shirt": "Shirts",
+  "accessory": "Accessories",
+  "postal": "Postal",
+  "meetUp": "Meet up",
+  "both": "Postal & Meet up",
+  "description": "Description",
+  "submit": "SUBMIT",
+  "namePlaceholder": "E.g. Nike Air Max, Adidas Ultraboost",
+  "pricePlaceholder": "E.g. 290, 3500.50",
+  "colorPlaceholder": "E.g. White, Black, Grey, Red, Blue",
+  "sizePlaceholder": "E.g. 38, 42, 7US, 10UK, S, M, L",
+  "descriptionPlaceholder": "Enter your product description",
+  "inputValueRequired": "This field is required.",
+  "inputNameInvalid": "This name is invalid.",
+  "inputPriceInvalid": "This price is invalid.",
+  "inputColorInvalid": "This color is invalid.",
+  "inputSizeExample": "This size is invalid.",
```

Should be naming in the same approach with other right?
```suggestion
  "inputColorInvalid": "This color is invalid.",
  "inputSizeInvalid": "This size is invalid.",
```